accessToken.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/*
+ * Creates an access token with VoiceGrant using your Twilio credentials.
+ */
+include('./vendor/autoload.php');
+include('./config.php');
+use Twilio\Jwt\AccessToken;
+use Twilio\Jwt\Grants\VoiceGrant;
+
+// Use identity and room from query string if provided
+$identity = isset($_GET["identity"]) ? $_GET["identity"] : "identity";
```

Comment:
By default PHP supports both `GET` and `POST` without any specifier in the source. We are assuming the request is coming in as `GET` but we want all the endpoints to support both methods.